internal/venice-common/src/test/java/com/linkedin/venice/blob/ServerBlobFinderTest.java

```diff
@@ -3,6 +3,8 @@
 import static org.mockito.Mockito.doReturn;
 import static org.mockito.Mockito.mock;
 
+import com.linkedin.venice.blobtransfer.BlobPeersDiscoveryResponse;
```

Comment:
Unused import?

Reply:
No, its used, it also gives me an error if i delete it